bots/HACKING.md

```diff
@@ -0,0 +1,38 @@
+# Hacking on the Cockpit Bots
+
+These are automated bots and testing that works on the Cockpit project. This
+includes updating operating system images, bringing in changes from other
+projects, releasing Cockpit and more.
+
+## Environment for the bots
+
+The bots work in containers that are built in the [cockpituous](https://github.com/cockpit-project/cockpituous)
+repository. New dependencies should be added there in the ```tests/Dockerfile```
+file in that repository.
+
+## Invoking the bots
+
+ # The containers in the `cockpitous` repository, invoke the ```.tasks``` file 
```

Comment:
Drop comma after "repository". This line and the following two prefixed with `#`, which looks really strange in Markdown (headings) and in plaintext too. Triple ` are unusual, please use single `

Reply:
Fixed